<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowofBANK</class>
 <widget class="QMainWindow" name="MainWindowofBANK">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>562</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>720</width>
    <height>562</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindowofBANK</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel{
background-color:#2b547d;
}
QMessageBox QLabel {
    background-color: transparent; /* Makes text background transparent */
    color: black; /* Text color */
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>731</width>
      <height>521</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
background-color:rgb(199, 201, 213);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>0</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
background-color:#2b547d;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>440</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
background-color:#2b547d;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>201</width>
      <height>471</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:10px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>20</y>
      <width>491</width>
      <height>471</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
background-color:rgb(255, 255, 255);
border-radius:10px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>40</y>
      <width>371</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
background-color:white;
font: 700 20pt &quot;Arial&quot;;
color:#2b547d;}</string>
    </property>
    <property name="text">
     <string>  Welcome to Islamic Bank</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>90</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
background-color:rgb(255, 255, 255);
	font: 12pt &quot;Arial&quot;;

color:#2b547d;}</string>
    </property>
    <property name="text">
     <string>Don't Have an account yet?</string>
    </property>
   </widget>
   <widget class="QPushButton" name="signup_btn">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>80</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft Sans Serif</family>
      <pointsize>12</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color:#3499db;
color:white;
border-radius:15px;
padding:7px 14px;
}
QPushButton:hover{
background-color:rgb(41, 128, 185);
}
QPushButton:pressed{
background-color:rgb(31, 97, 141);
}</string>
    </property>
    <property name="text">
     <string>SIGN UP</string>
    </property>
   </widget>
   <widget class="QPushButton" name="login_btn">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>400</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft Sans Serif</family>
      <pointsize>12</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color:#2b547d;
color:white;
border-radius:15px;
padding:7px 14px;
}
QPushButton:hover{
background-color:rgb(41, 128, 185);
}
QPushButton:pressed{
background-color:rgb(31, 97, 141);
}</string>
    </property>
    <property name="text">
     <string>LOG IN</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="username_input">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>160</y>
      <width>371</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{

background-color: rgb(255, 255, 255);
color:rgb(43, 84, 125);
border:2px solid rgb(43, 84, 125);
padding:5px;
padding-left:45px;
}
QLineEdit:hover{
border:2px solid rgb(29, 61, 92);
background-color:rgb(226, 230, 235);
}
QLineEdit:focus{
border:2px solid rgb(0, 0, 119);
background-color:white;
outline:none;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>username</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="gmail_input">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>230</y>
      <width>371</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{

background-color: rgb(255, 255, 255);
color:rgb(43, 84, 125);
border:2px solid rgb(43, 84, 125);
padding:5px;
padding-left:45px;
}
QLineEdit:hover{
border:2px solid rgb(29, 61, 92);
background-color:rgb(226, 230, 235);
}
QLineEdit:focus{
border:2px solid rgb(0, 0, 119);
background-color:white;
outline:none;
}</string>
    </property>
    <property name="placeholderText">
     <string>gmail</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="password_input">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>300</y>
      <width>371</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{

background-color: rgb(255, 255, 255);
color:rgb(43, 84, 125);
border:2px solid rgb(43, 84, 125);
padding:5px;
padding-left:45px;
}
QLineEdit:hover{
border:2px solid rgb(29, 61, 92);
background-color:rgb(226, 230, 235);
}
QLineEdit:focus{
border:2px solid rgb(0, 0, 119);
background-color:white;
outline:none;
}</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::EchoMode::Password</enum>
    </property>
    <property name="placeholderText">
     <string>password</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>140</y>
      <width>181</width>
      <height>151</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="images.qrc">:/new/prefix1/images/wmremove-transformed.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>460</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="images.qrc">:/new/prefix1/images/wmremove-transformed.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>10</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="images.qrc">:/new/prefix1/images/wmremove-transformed.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>310</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	background-color: none;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="images.qrc">:/new/prefix1/images/padlock.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>170</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	background-color: none;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="images.qrc">:/new/prefix1/images/profile.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>240</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	background-color: none;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="images.qrc">:/new/prefix1/images/mail.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>label</zorder>
   <zorder>label_4</zorder>
   <zorder>label_5</zorder>
   <zorder>label_6</zorder>
   <zorder>label_7</zorder>
   <zorder>signup_btn</zorder>
   <zorder>login_btn</zorder>
   <zorder>password_input</zorder>
   <zorder>label_2</zorder>
   <zorder>label_11</zorder>
   <zorder>label_12</zorder>
   <zorder>username_input</zorder>
   <zorder>gmail_input</zorder>
   <zorder>label_3</zorder>
   <zorder>label_13</zorder>
   <zorder>label_8</zorder>
   <zorder>label_9</zorder>
   <zorder>label_10</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>720</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>0</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="acceptDrops">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
